apps/allocations/app/components/App/BudgetDetail.js

```diff
@@ -38,6 +40,15 @@ const displayCurrency = (amount, decimals) => {
     .toLocaleString()
 }
 
+const processTime = ms => {
+  const days = Math.round(ms/MS_PER_DAY)
+  if(days%7 === 0){
+    const weeks = days/7
+    return `per ${weeks === 1 ? 'week' : `${String(weeks)} weeks`}`
+  }
+  return `per ${days === 1 ? 'day' : `${String(days)} days`}`
```

Comment:
I am in favour of having this calculated already in the state or app-reducer instead of each time the component is rendered, so we can eventually use it elsewhere instead of forcing us to repeat these calculations each time we want to use them in the UI, so I think defining this locally is not the optimal solution.

Reply:
Anyway we can have it this way if we are sure this is not needed elsewhere for now 👍